cron.py

```diff
@@ -1221,6 +1223,7 @@ def clear_users_scheduled_to_be_deleted(dry_run=False):
         )
         if dry_run:
             continue
+        EventDispatcher.send_event(user, EventContent(user_deleted=UserDeleted()))
         User.delete(user.id)
```

Comment:
Why not put it inside the `User.delete` method?

Reply:
Because it didn't exist beforehand, but I've created it now and moved the logic there